host-configs/tpls.cmake

```diff
@@ -53,7 +53,9 @@ if(EXISTS ${GEOSX_TPL_DIR}/vtk)
 endif()
 
 if(EXISTS ${GEOSX_TPL_DIR}/fmt)
-  set(FMT_DIR ${GEOSX_TPL_DIR}/fmt CACHE PATH "" FORCE)
+#  set(FMT_DIR ${GEOSX_TPL_DIR}/fmt CACHE PATH "" FORCE)
+  set(FMT_DIR ${GEOSX_TPL_DIR}/chai CACHE PATH "" FORCE)
```

Comment:
Is FMT_DIR pointing to chai ?

Reply:
yes. there is a conflict. A lot of the TPL's have `fmt` integrated, and `umpire` which is included in `CHAI` changed the way they include it so now there can be a conflict....it is annoying to say the least.